airflow/utils/dates.py

```diff
@@ -225,3 +226,14 @@ def days_ago(n, hour=0, minute=0, second=0, microsecond=0):
         second=second,
         microsecond=microsecond)
     return today - timedelta(days=n)
+
+
+def to_iso(execution_date):
+    """ Parse execution date to iso format. """
+    iso = execution_date
+    try:
+        iso = execution_date.isoformat()
+    except AttributeError:
```

Comment:
Why would this throw AttributeError? Is it because execution_date can have different types? In that case I think cleaner would be to have multiple methods like <type>_to_iso instead of just one to_iso method.